pyanaconda/modules/payloads/payloads.py

```diff
@@ -238,7 +232,7 @@ def teardown_with_tasks(self):
         if self.active_payload:
             tasks += self.active_payload.tear_down_with_tasks()
 
-        if self._flatpak_side_payload:
-            tasks += self._flatpak_side_payload.tear_down_with_tasks()
+        if self.active_payload.side_payload:
```

Comment:
Shouldn't we check for active_payload here as well (or return based on the preivous condition?, or rather add the check at the beginning as install_with_tasks() and some other methods?